apps/backend/api/graphql/makeModels.js

```diff
@@ -60,6 +60,42 @@ export default function makeModels (userId, isAdmin, apiClient) {
       fetchMany: () => CommonRole.fetchAll()
     },
 
+    ContextWidget: {
+      model: ContextWidget,
+      attributes: [
+        'id',
+        'title',
+        'type', 
+        'order',
+        'visibility',
+        'view',
+        'icon',
+        'created_at',
+        'updated_at',
+        'highlightNumber',
+        'secondaryNumber'
```

Comment:
to discuss

Reply:
Yeah, these are still a bit of a mystery in terms of what they need to represent. I know you asked those questions via figma comments